src/Components/MediaWrapper/ImageWrapper.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { useEffect } from "react";
+import { imageDuration } from "..";
+import styles from "./ImageWrapper.module.css";
+
+type Props = {
+  file: File;
+  changeImage: () => void;
+  duration: number;
+};
+// future: add media can be optional
+// if no add media, then do ot set timeout
+export const ImageWrapper: React.FC<Props> = ({ file, changeImage, duration }) => {
+  useEffect(() => {
+    const timeout = setTimeout(() => {
```

Comment:
can you use `useTimeout` from 'beautiful-react-hooks' instead?

Reply:
I'm not using beautiful-react-hooks anywhere inside the app anymore, would it still be beneficial to use an extra library?

Reply:
ah no, sorry, I thought you still were 😁